terraform/modules/core/core.tf

```diff
@@ -453,15 +453,20 @@ resource "aws_autoscaling_group" "ecs-autoscaling-group" {
   ]
 
   depends_on = ["aws_vpc.vpc", "aws_launch_configuration.ecs-launch-configuration", "aws_security_group.default", "aws_security_group.ecs"]
+
+
+  lifecycle {
+    ignore_changes = ["desired_capacity"]
```

Comment:
Ensure every time we deploy, we don't set the EC2 instance count back down to what's configured and change it from what the infrastructure load has calculated. This would undermine the autoscaling and make us more likely to serve 503.